### PR TITLE
Update install instructions for new clone path

### PR DESCRIPTION
**Pull request by riking**
(Keeping previous import path)

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/go-toxcore-c/17)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ The golang bindings for libtoxcore
 
 ### Installation
 
+    mkdir -p $(go env GOPATH)/github.com/kitech
+    ( cd $(go env GOPATH)/github.com/kitech; git clone https://github.com/TokTok/go-toxcore-c.git go-toxcore )
+    # fetch libtoxcore if necessary
+    # see https://github.com/irungentoo/toxcore/blob/master/INSTALL.md
     go get github.com/kitech/go-toxcore
```
Comment (iphydf) [inline README.md]: Why not change this to the new URL?

Comment (riking) [reply README.md]: I wasn't sure what the correct canonical URL was - is it this repository? The earlier `cd` commands place this repository in the old path.

Comment (iphydf) [reply README.md]: @kitech thoughts?

As far as I understand, we'll have the toktok repo as the canonical one for the core API. kitech wants to work on a higher level API. TokTok will maintain the low level one.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ The golang bindings for libtoxcore
 
 ### Installation
 
-    go get github.com/kitech/go-toxcore
+    # fetch libtoxcore if necessary
+    # see https://github.com/irungentoo/toxcore/blob/master/INSTALL.md
```
Comment (iphydf) [inline README.md]: TokTok/c-toxcore instead of irungentoo/toxcore

**Comment by riking** [comment]
Updated.

**Pull request merged**